.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
You should remove this file, it looks like it got accidentally included in your commit,
If youre using git in the terminal check this out
http://stackoverflow.com/questions/12481639/remove-files-from-git-commit